tests/CMakeLists.txt

```diff
@@ -47,6 +47,13 @@ function(add_one_test)
             -Werror
     )
 
+    # Include directories.
+    target_include_directories(
+        ${EXECUTABLE_NAME}
+        PRIVATE
+            ${CMAKE_SOURCE_DIR}
+    )
```

Comment:
This shouldn't be needed, right ? Since we're explicitly linking to `Kokkos::utils` a few lines below, to ensure that the target has all the needed include files.

Reply:
OK. No longer needed now that the matcher p has disappeared.